AnnotatorCore.py

```diff
@@ -1504,13 +1505,17 @@ def getimplications(oncokbdata, implication_type, levels, implications):
 
 class GenomicChangeQuery:
     def __init__(self, chromosome, start, end, ref_allele, var_allele, cancertype, reference_genome=None):
+        if chromosome is not None:
+            chromosome = chromosome.strip()
+            if chromosome.startswith('chr'):
+                chromosome = chromosome.strip()[3:]
```

Comment:
```suggestion
            chromosome = chromosome.strip()
            if chromosome.startswith('chr'):
                chromosome = chromosome[3:]
```
Redundant `.strip()`